integration-cli/docker_cli_help_test.go

```diff
@@ -27,9 +23,10 @@ func TestMainHelpWidth(t *testing.T) {
 	for _, line := range lines {
 		if len(line) > 80 {
 			t.Fatalf("Line is too long(%d chars):\n%s", len(line), line)
+
 		}
 		if home != "" && strings.Contains(line, home) {
-			t.Fatalf("Line should use ~ instead of %q:\n%s", home, line)
+			t.Fatalf("Line should use home shortcut instead of %q:\n%s", home, line)
```

Comment:
Just an idea, but should we just create the following (either that or a `const` conditioned by a build tag, whatever):

```
func homeDirectorySpecifier() string {
    if runtime.GOOS == "windows" {
        return "%USERPROFILE%"
    }
    return "~"
}
```

That would help us keep the same output as we have today, *and* could be use below for `PrintDefaults()` implementation.